docs/advanced-guide/handling-file/page.md

```diff
@@ -15,13 +43,13 @@ err := ctx.File.Mkdir("my_dir",os.ModePerm)
 
 To create subdirectories as well
 ```go
-err := c.File.MkdirAll("my_dir/sub_dir", os.ModePerm)
+err := ctx.File.MkdirAll("my_dir/sub_dir", os.ModePerm)
```

Comment:
I wouldn't use ctx as it's connoted for context.Context variable

Reply:
But if someone is using gofr, we have actually wrapped the context.Context in gofr's context.
Won't that be better for users to understand? as they won't be using context.Context